hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/common/HdfsServerConstants.java

```diff
@@ -366,6 +366,7 @@ public void write(DataOutput out) throws IOException {
       "security.hdfs.unreadable.by.superuser";
   String XATTR_ERASURECODING_POLICY =
       "system.hdfs.erasurecoding.policy";
+  String SNAPSHOT_XATTR_NAME = "system.hdfs.snapshot";
```

Comment:
The xattr name should contain "deleted".  How about "system.hdfs.snapshot.deleted"?